docs/getting-started/setup/editor-types.md

```diff
@@ -66,7 +67,7 @@ See an {@link examples/builds/balloon-block-editor example of the balloon block
 
 The Decoupled editor is named for its unique structure, where the toolbar and editing area are separate elements. This design allows for greater flexibility and customization, making it suitable for a wide range of applications beyond just classic WYSIWYG editing.
 
-The most popular use case for the Decoupled editor is the “document editor”, similar to large editing packages such as Google Docs or Microsoft Word. It works best for creating documents, which are usually later printed or exported to PDF files.
+The most popular use case for the Decoupled editor is the “document editor” similar to large editing packages such as Google Docs or Microsoft Word. It works best for creating documents, which are usually later printed or exported to PDF files.
```

Comment:
I wasn't sure either, so I checked, and the comma should probably land here. I think the punctuation mark order is now correct.
```suggestion
The most popular use case for the Decoupled editor is the “document editor,” similar to large editing packages such as Google Docs or Microsoft Word. It works best for creating documents, which are usually later printed or exported to PDF files.
```